h2o-genmodel/src/main/java/hex/genmodel/algos/ensemble/StackedEnsembleMojoModel.java

```diff
@@ -21,9 +38,31 @@ private static double logit(double p) {
         return  x == 0 ? -19 : Math.max(-19, Math.log(x));
     }
 
-    private static void logitTransformRow(double[] basePreds){
+    private static double[] logitTransformRow(double[] basePreds){
```

Comment:
This change in the API might be useful if we will decide to use percentile rank combined with logit transform (cbinded together) so we output more "columns" than there is base models.